app/assets/javascripts/time_jump.js

```diff
@@ -1,33 +1,37 @@
 document.addEventListener('DOMContentLoaded', function() {
```

Comment:
any reason why we're handling the parsing of the date in the js? I think we might be able to let rails handle it in the action we hit. see `parse_date` in `ApplicationController`

Reply:
this was how the team that built it did it. I think it is fine to leave for now since we need to unblock that team, but a good point for later refactor

Reply:
In the new bs4 iteration, we're letting rails handle it, because theoretically it should already be formatted correctly with Rails catching any errors.

Most of the changes are to accommodate the previous way we were submitting a date here, which was a different field type. We're also doing a bit of displaying with the date, which the old version needs to format prior to displaying.

For bs4, we're basically just checking that it's parsable, then passing it to Rails, which does parse the date also. I think they mainly went through all the hassle just to display it in the modal prior to submitting it.